language/diem-framework/compiled/src/lib.rs

```diff
@@ -148,3 +163,16 @@ pub fn env_stdlib_modules() -> StdLibModules {
 pub fn use_compiled() -> bool {
     std::env::var(NO_USE_COMPILED).is_err()
 }
+
+pub fn old_stdlib_modules(hash: GitHash) -> Result<(Vec<Vec<u8>>, Vec<CompiledModule>)> {
```

Comment:
This is the newly introduced api. One downside tho is that this api cannot use `StdLibModules` because of the lifetime issue.

Reply:
The rest of the diff looks good. Could `StdlibModules` own its `CompiledModule` instances?